packages/utils-create-server/src/index.ts

```diff
@@ -112,20 +118,26 @@ export class Server {
         return this.actionWrapper('stop');
     }
 
+    public set port(port: number) {
+        this._port = port;
+    }
+
     public get port() {
         return this._port;
     }
 
     public configure(configuration: ServerConfiguration) {
         return this.actionWrapper('configure', configuration);
     }
+}
 
+export class Server {
```

Comment:
IMO, the naming is a little bit confusing, you are importing a Server renamed to ServerProcess, and then you are exporting here another Server that looks to be a factory for `Server` and `IndependentServer`, plus other functionality.

Reply:
`Server` and `IServer` are the only pieces seen from the outside. As you've seen `Server` is a factory of `IServer` that will decide what to use depending on the environment or the user's configuration (`Independent` if not in ci).